src/argilla/client/feedback/integrations/huggingface/card/__init__.py

```diff
@@ -15,4 +15,7 @@
 from ._dataset_card import ArgillaDatasetCard
 from ._parser import size_categories_parser
 
-__all__ = ["ArgillaDatasetCard", "size_categories_parser"]
+__all__ = [
+    "ArgillaDatasetCard",
+    "size_categories_parser",
+]
```

Comment:
Why this change here?

Reply:
When I added the model card, the classes currently in `model_card/__init__.py` were here. Then I moved the model card to its own module but forgot to let this import as it was originally.

Reply:
@plaguss could you add a small PR to fix the comments from @alvarobartt

Reply:
@davidberenstein1957 here it is: https://github.com/argilla-io/argilla/pull/3983